ios/Classes/TiNfcVTagTechnology.m

```diff
@@ -9,4 +9,701 @@
+                                                                         @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                                       }];
+                                                                }];
+}
+
+- (void)extendedGetMultipleBlockSecurityStatusWithRequestFlag:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  [[self.tagProxy asNFCISO15693Tag] extendedGetMultipleBlockSecurityStatusWithRequestFlag:requestFlags
+                                                                               blockRange:NSMakeRange(0, 8)
+                                                                        completionHandler:^(NSArray<NSNumber *> *_Nonnull securityStatus, NSError *_Nullable error) {
+                                                                          if (![self _hasListeners:@"didExtendedGetMultipleBlockSecurityStatusWithRequestFlag"]) {
+                                                                            return;
+                                                                          }
+                                                                          [self fireEvent:@"didExtendedGetMultipleBlockSecurityStatusWithRequestFlag"
+                                                                               withObject:@{
+                                                                                 @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
+                                                                                 @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
+                                                                                 @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                                               }];
+                                                                        }];
+}
+
+- (void)extendedWriteMultipleBlocksWithRequestFlags:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  NSArray *dataBlock = [[args firstObject] valueForKey:@"dataBlock"];
+  unsigned int dataValue[dataBlock.count];
+  for (int i = 0; i < dataBlock.count; i++) {
+    dataValue[i] = [dataBlock[i] unsignedIntValue];
+  }
+  NSData *data = [[NSData alloc] initWithBytes:dataValue length:dataBlock.count];
+  NSArray *dataBlocks = [[NSArray alloc] initWithObjects:data, nil];
+
+  [[self.tagProxy asNFCISO15693Tag] extendedWriteMultipleBlocksWithRequestFlags:requestFlags
+                                                                     blockRange:NSMakeRange(0, 8)
```

Comment:
Why using hard value? As per apple doc "The range of blocks.  Valid start index range is 0 to 65535 inclusively.  Valid length is 1 to 65536 inclusively."